test/expect-tests/path/dune

```diff
@@ -0,0 +1,15 @@
+;; these tests are separate b/c they set the build dir
```

Comment:
Maybe we could set the build dir globally in the tests? i.e., instead of calling `Path.set_root ...` explicitly in the .ml file of the test, we could do: `Common.init_paths ()`. Then they wouldn't have to be separate.